experimenter/experimenter/jetstream/models.py

```diff
@@ -70,17 +80,12 @@ class Group:
         METRIC_GROUP[metric] = group
 
 
-class JetstreamDataPoint(BaseModel):
-    lower: float = None
-    upper: float = None
-    point: float = None
-    metric: str = None
-    branch: str = None
-    statistic: str = None
-    window_index: str = None
-    comparison: str = None
-    segment: str = Segment.ALL
-    analysis_basis: str = AnalysisBasis.ENROLLMENTS
+class JetstreamDataPoint(JetstreamStatisticResult):
+    """
+    Same as the mozilla-nimbus-schemas `Statistic` but sets a default analysis_basis.
```

Comment:
Python docstrings should begin on the `"""` line.

Reply:
Yea, but then it goes over the 90-character line length. I guess I could edit the sentence to make it a few characters shorter.